test/pkg/environment/common/expectations.go

```diff
@@ -924,3 +926,17 @@ func (env *Environment) GetDaemonSetOverhead(np *corev1beta1.NodePool) v1.Resour
 		return p, true
 	})...)
 }
+
+func (env *Environment) EventuallyExpectNodeClassStatusCondition(nodeClass *v1beta1.EC2NodeClass, condition string, status bool, message string) {
+	GinkgoHelper()
+
+	Eventually(func(g Gomega) {
+		temp := &v1beta1.EC2NodeClass{}
+		g.Expect(env.Client.Get(env, client.ObjectKeyFromObject(nodeClass), temp)).To(Succeed())
+		statusCondition := temp.StatusConditions().Get(condition)
+		g.Expect(statusCondition.IsTrue()).To(Equal(status))
+		if len(message) != 0 {
+			g.Expect(statusCondition.Message).To(Equal(message))
+		}
```

Comment:
I would expect that if we're expecting an empty message, we can just always assert that. 
```suggestion
		g.Expect(statusCondition.IsTrue()).To(Equal(status))
	        g.Expect(statusCondition.Message).To(Equal(message))
```